common/doordeck.js

```diff
@@ -32,10 +30,10 @@ const doordeckInit = function (authToken) {
   })
 }
 
-const unlock = function (deviceId) {
+const unlock = function (baseUrl, deviceId) {
   return new Promise (function (resolve, reject) {
     libSodium.ready.then(function () {
-      deviceOperation.unlock(deviceId, 7).then(response => {
+      deviceOperation.unlock(baseUrl, deviceId, 7).then(response => {
```

Comment:
The server ignores any unlock time provided by the client in the unlock calls, so you can remove the `7` and any code related to custom unlock timers.

Reply:
these aren't being used so I'm deleting them